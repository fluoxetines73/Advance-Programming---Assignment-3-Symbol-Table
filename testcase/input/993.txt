BEGIN
INSERT d string
ASSIGN d 'UPPERlower' random
INSERT f_FF string
INSERT z string
LOOKUP f_FF
PRINT
END