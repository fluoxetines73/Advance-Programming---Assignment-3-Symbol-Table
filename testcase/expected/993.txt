Invalid: ASSIGN d 'UPPERlower' random